templates/org/aac/aac-architecture-guide-content.md

```diff
@@ -44,12 +44,19 @@ Other contributors: > Include contributing (but not primary) authors, major edit
 
 ## Next steps
```

Comment:
The purpose of this change is to resolve confusion about what kind of content each section should include. One of the Aquent editors brought up this issue. To resolve the confusion, I pasted the section info from the aac-article-title-content.md template, where the purpose of the sections is presumably the same.